apps/main/src/lend/store/createOhlcChartSlice.ts

```diff
@@ -308,35 +309,40 @@ const createOhlcChart = (set: SetState<State>, get: GetState<State>) => ({
       const network = networks[chainId].id.toLowerCase()
 
       try {
-        const llammaOhlcFetch = await fetch(
-          `https://prices.curve.fi/v1/lending/llamma_ohlc/${network}/${poolAddress}?agg_number=${interval}&agg_units=${timeUnit}&start=${start}&end=${end}`,
-        )
-        const llammaOhlcResponse: LlammaOhlcApiResponse = await llammaOhlcFetch.json()
+        const ohlc = await getOHLC({
+          endpoint: 'lending',
+          chain: network as Chain,
+          llamma: poolAddress as Address,
+          interval,
+          units: timeUnit as Parameters<typeof getOHLC>[0]['units'],
+          start,
+          end,
+        })
 
         let volumeArray: VolumeData[] = []
         let baselinePriceArray: LlamaBaselinePriceData[] = []
         let oraclePriceArray: OraclePriceData[] = []
         let ohlcDataArray: LpPriceOhlcDataFormatted[] = []
 
-        for (const item of llammaOhlcResponse.data) {
+        for (const item of ohlc) {
           volumeArray.push({
-            time: convertToLocaleTimestamp(item.time) as UTCTimestamp,
+            time: item.time as UTCTimestamp,
             value: item.volume,
             color: item.open < item.close ? '#26a69982' : '#ef53507e',
           })
 
           baselinePriceArray.push({
-            time: convertToLocaleTimestamp(item.time) as UTCTimestamp,
-            base_price: item.base_price,
+            time: item.time as UTCTimestamp,
+            base_price: item.priceBase,
           })
 
           oraclePriceArray.push({
-            time: convertToLocaleTimestamp(item.time) as UTCTimestamp,
-            value: item.oracle_price,
+            time: item.time as UTCTimestamp,
+            value: item.priceOracle,
```

Comment:
priceOracle sounds like the address of the oracle.
oraclePrice seems like the correct name here